x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/hooks/use_entity_engine_status.ts

```diff
@@ -15,12 +17,25 @@ export const ENTITY_STORE_ENGINE_STATUS = 'ENTITY_STORE_ENGINE_STATUS';
 interface Options {
   disabled?: boolean;
   polling?: UseQueryOptions<ListEntityEnginesResponse>['refetchInterval'];
+  withComponents?: boolean;
 }
 
 interface EngineError {
   message: string;
 }
 
+export const useEntityEngineStatusNew = (opts: Options = {}) => {
```

Comment:
Temporary while waiting for https://github.com/elastic/kibana/pull/199762 to be merged

Reply:
done!